2024-frontend/src/views/DiagnosticTunnel/index.vue

```diff
@@ -2,13 +2,20 @@
 import keyMeasures from "@/data/key-measures.json"
 // TODO: sort out **/index.vue imports so don't have to add index.vue
 import WasteMeasureSteps from "./WasteMeasureSteps/index.vue"
```

Comment:
```suggestion
import WasteMeasurementSteps from "./WasteMeasurementSteps/index.vue"
```
suite à notre discussion mattermost, autant utiliser "Measurement" partout si on part sur ce wording :pray:

Reply:
ici, j'utilise le mot "Measure" tel qu'il est utilisé dans l'ancien front, pour referencé plutôt le mesure anti-gaspi de la loi et pas tellement lié au modèle. Après, je vois comment c'est confusant. Si tu veux on pourrait appeler pour decider le meilleur nom. (Pour contexte, j'utilise le mot Measure dans l'ancien front entre les 5 mesures pour avoir un nommage coherent entre les 5 étapes du tunnel bilan. Avec la discussion de #4207 je sais pas si on va couper le tunnel diagnostic du bilan complètement ou non...)